doc/lightning-listpeers.7.md

```diff
@@ -100,6 +100,8 @@ The objects in the *channels* array will have at least these fields:
   a number followed by a string unit.
 * *total\_msat*: A string describing the total capacity of the channel;
   a number followed by a string unit.
+* *fee_base*: The applied routing fees for this half channel as msat string.
+* *fee_ppm*: The applied proportional ppm fees for this half channel as a number.
```

Comment:
Similarly "the routing fee we will charge for forwards going out over this channel, in millisatoshi for every 1 million millisatoshi ('parts per million') of payment size."

Reply:
Also suggest adding `lightning-setchannelfee` in the "SEE ALSO" section at the end of the file.

Reply:
Ack, but I skipped the 'millisatoshi' part, as this is not relevant for a proportional value if its 'millisatoshi', 'satoshi', 'bitcoin' or 'finney'. Acutally the `setchannelfee` doc has the same issue and quotes ppm as 'in satoshi' :D ... however I wrote this manpage two years ago.